scan-local.sh

```diff
@@ -22,11 +22,30 @@ docker run -d --name "$name" --volumes-from "$name-data" repo-info:local > /dev/
 
 echo '# `'"$image"'`'
 
+size="$(
+	docker inspect -f '{{ .VirtualSize }}' "$image" | awk '{
+		oneKb = 1000;
+		oneMb = 1000 * oneKb;
+		oneGb = 1000 * oneMb;
+		if ($1 >= oneGb) {
+			printf "~ %.2f Gb", $1 / oneGb
+		} else if ($1 >= oneMb) {
+			printf "~ %.2f Mb", $1 / oneMb
+		} else if ($1 >= oneKb) {
+			printf "~ %.2f Kb", $1 / oneKb
```

Comment:
To put that another way, I don't want the bug reports saying that `docker images` shows X but we show Y so we're Clearly Wrong. :innocent: